rasa/utils/train_utils.py

```diff
@@ -471,7 +473,13 @@ def create_common_callbacks(
     Returns:
         A list of callbacks.
     """
-    import tensorflow as tf
+    try:
+        import tensorflow as tf
+    except ModuleNotFoundError:
+        raise MissingDependencyException(
+            f"Tensorflow does not seem to be installed, but it is needed for this "
+            f"{DOCS_BASE_URL}/installation"
```

Comment:
Could you make the error message more descriptive (example usage of `MissingDependencyException can be found in [cli/x.py](https://github.com/RasaHQ/rasa/blob/1aa1b6c93ba2603064737814f9fcdce764af40d9/rasa/cli/x.py#L470) and [nlu/components.py](https://github.com/RasaHQ/rasa/blob/1aa1b6c93ba2603064737814f9fcdce764af40d9/rasa/nlu/components.py#L87))? Also, the general documentation page linked in the message doesn't mention tensorflow, could you therefore add something along the lines of `For an overview of how to install Rasa, please visit URL`.

Reply:
Kept it simple :)